src/app/components/property-details/property-details.component.ts

```diff
@@ -40,20 +41,26 @@ export class PropertyDetailsComponent implements OnInit, OnDestroy {
   clientCity: string;
   purchasePlan: string;
   subscribe: Subscription[] = [];
-
+  displayedModal: boolean = false;
+  isMine: boolean = false;
```

Comment:
I am suggesting we use
```suggestion
  isMine = false;
```
since we get a communication from angular saying

> Type boolean trivially inferred from a boolean literal, remove type annotation

which simply means TS understands that false is a boolean